PokemonRampUp/Model/Move.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Move.swift
+//  PokemonRampUp
+//
+//  Created by Thushen Mohanarajah on 2023-02-14.
+//
+
+import Foundation
+
+// MARK: - Move
+struct Move: Decodable {
+    let move: NameAndURL
```

Comment:
I would rename NameAndURL to move

Reply:
I changed it to something similar . I couldn't call it move because the struct itself is called Move